index.html

```diff
@@ -61,10 +62,198 @@ <h3>LET’S CONNECT</h3>
                 </ul>
 
             </div>
+                  <!-- start of work section -->
+
+<div class="work-container">
+    <div class="img-snapshot1">
+    <img src="images/Snapshoot1.png" alt="">
+    </div>
+    <div class="tonic1">
+        <h2>Tonic</h2>
+    </div>
+    <div class="frame">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
```

Comment:
- [ ] Please consider not using images for the project tags and building them yourself according to the Figma design. (kindly do that for each card)

Reply:
- [ ] Nice Working on your tags, Like the previous reviewer said kindly consider not using images, for better practice and use of HTML semantics.